lib/Driver/ToolChains.cpp

```diff
@@ -242,6 +242,8 @@ void ToolChain::addCommonFrontendArgs(const OutputInfo &OI,
   inputArgs.AddLastArg(arguments,
                        options::OPT_emit_fine_grained_dependency_sourcefile_dot_files);
   inputArgs.AddLastArg(arguments, options::OPT_package_description_version);
+  inputArgs.AddLastArg(arguments, options::OPT_locale);
```

Comment:
This can handle for example from "pt-br" and "pt-pt"? Or this intentionally meant to be that way to only handle "pt" or "br" in those cases?

Reply:
Ahh... sorry, nevermind I just get it :)